src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -1399,6 +1399,8 @@ public sealed override async Task<HttpResponseMessage> SendAsync(HttpRequestMess
                     }
                     else
                     {
+                        await responseHeadersTask;
+
```

Comment:
Shouldn't the await be moved down to below the ContinueWith and nulling out below?